rotkehlchen/externalapis/cryptocompare.py

```diff
@@ -843,6 +843,13 @@ def query_historical_price(
             )
 
         log.debug('Got historical price from cryptocompare', from_asset=from_asset, to_asset=to_asset, timestamp=timestamp, price=price)  # noqa: E501
+        GlobalDBHandler().add_historical_prices(entries=[HistoricalPrice(
+            from_asset=from_asset,
+            to_asset=to_asset,
+            source=HistoricalPriceOracle.CRYPTOCOMPARE,
+            timestamp=timestamp,
+            price=price,
+        )])
```

Comment:
Added this here to save prices from cryptocompare queried using this method. For what I saw it was missing

Reply:
Hmm should be fine. I think we just did cache in the sense of searching all historical queries in histohour before, but this should also work fine.